Add -n and --no-mtab arguments. As we do not write an mtab file, this
is a trivial patch that just ignores this argument to be more
compatible with Linux mount.

* utils/mount.c (argp_opts): Add -n and --no-mtab.
(parse_opt): Do nothing on 'n'.
---
 utils/mount.c |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/utils/mount.c b/utils/mount.c
index 8b059c2..ea30f7a 100644
--- a/utils/mount.c
+++ b/utils/mount.c
@@ -54,6 +54,7 @@ static const struct argp_option argp_opts[] =
   {"update", 'u', 0, 0, "Flush any meta-data cached in core"},
   {"remount", 0, 0, OPTION_ALIAS},
   {"verbose", 'v', 0, 0, "Give more detailed information"},
+  {"no-mtab", 'n', 0, 0, "Do not update /etc/mtab"},
   {0, 0}
 };
 
@@ -110,6 +111,10 @@ parse_opt (int key, char *arg, struct argp_state *state)
 	}
       break;
 
+    case 'n':
+      /* do nothing */
+      break;
+
     case ARGP_KEY_ARG:
       if (mountpoint == 0)	/* One arg: mountpoint */
 	mountpoint = arg;
-- 
1.7.10.4


